### PR TITLE
Update auth library version in sample

### PR DESCRIPTION
**Pull request by veronikapetruskova**

### CODE REVIEW

**Comment by veronikapetruskova** [comment]
@mdelolmo

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,12 @@ Change Log
 
 ## Version 1.1.1
 
-_2018-02-28_
+_2019-12-05_
+
+* Breaking changes: Rename classes from AuthenticationClassName to AuthorizationClassName
```
Comment (mdelolmo) [inline CHANGELOG.md]: 👍

File: auth-sample/build.gradle [hunk 2]
```diff
@@ -53,7 +53,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.spotify.android:auth:1.1.0'
+    implementation 'com.spotify.android:auth:1.1.1'
```
Comment (mdelolmo) [inline auth-sample/build.gradle]: I'd almost say this should be 1.2.0, since it contains breaking changes. What do you think?

Comment (veronikapetruskova) [reply auth-sample/build.gradle]: yes, definitely.

**Comment by veronikapetruskova** [comment]
@mdelolmo

**Pull request merged**